src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageSolutionDetailControlModel.cs

```diff
@@ -94,6 +114,8 @@ private void UpdateInstalledVersions()
                 }
             }
 
+            IsRequestedVisible = _projects.Any(p => p.NuGetProject.ProjectStyle.Equals(NuGet.ProjectModel.ProjectStyle.PackageReference));
```

Comment:
Could this boolean be set inside your loop when setting RequestedVersion? 
Or is it possible to have a `null` installedVersion but that project still should show Requested versions?

Reply:
It should show the Requested versions column, this column depends on the project type, but I can move this inside the loop

Reply:
I suggested another refactoring. I meant that your loop already does this work, so not sure we need to call `.Any` again?